src/main/java/ManagmentSystem.java

```diff
@@ -1,134 +1,20 @@
-import java.sql.Date;
 import java.util.ArrayList;
 
+/**
+ * 
+ * This class will be an Institute Management System
+ *
+ */
 public class ManagmentSystem {
 
 	public static void main(String[] args) {
-
+		
+       ArrayList<Department> insituDepartments = new ArrayList<Department>();
+       ComputerScienceDepartment csDep = null;
```

Comment:
we should be instantiating (new ComputerScienceDepartment()) here and then put it in the ArrayList to tell the viewers that its composition. If both are passed as null then that means None is mandatory for the existence of Department.